packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -118,7 +119,7 @@ export const getNotificationPageData = async (
         if (imageUrl === undefined) {
             throw new Error('Unable to unfurl image');
         }
-    } else {
+    } else if (format === SchedulerFormat.CSV) {
```

Comment:
you could also do a `switch` here with `default` being `assertUnreachable`.

Did you look at what's stored in db ?  

Image was first, then we introduced CSV, perhaps IMage should be the default. Perhaps those customers created the schedule before we added CSV, we could check created_at on db

Reply:
Image is actually set by default as per this migration file: https://github.com/lightdash/lightdash/blob/main/packages/backend/src/database/migrations/20230216141637_add_scheduler_format.ts#L8

So I really wonder what happened for users to experience this... 
I will change this to a `switch`, thanks for that!